connect/runtime/src/main/java/org/apache/kafka/connect/runtime/TransformationChain.java

```diff
@@ -40,6 +44,8 @@ public R apply(R record) {
         for (final Transformation<R> transformation : transformations) {
             final R current = record;
 
+            log.trace("Applying SMT transformation: {}, record: {}",
```

Comment:
These log messages really make most of the individual log messages within the SMTs less valuable, since the record will include the key schema, key, value schema, and value.

I've highlighted some of the log messages in individual SMTs that I think are worthwhile.

Also: 
```suggestion
            log.trace("Applying SMT transformation {} to {}",
```

Reply:
Also, I just looked at `ConnectRecord.toString()`, and it does _not_ print the schemas. I wonder if it's worth changing that to include whether the key schema and value schema are null; e.g.,
```
    @Override
    public String toString() {
        return "ConnectRecord{" +
                "topic='" + topic + '\'' +
                ", kafkaPartition=" + kafkaPartition +
                ", keySchema=" + (keySchema != null ? "..." : "null") +
                ", key=" + key +
                ", valueSchema=" + (valueSchema != null ? "..." : "null") +
                ", value=" + value +
                ", timestamp=" + timestamp +
                ", headers=" + headers +
                '}';
    }
```